### PR TITLE
Admin form firebase connection

### PR DESCRIPTION
**Pull request by J4yM1l**
admin form can now add data to firestore.

### CODE REVIEW

**Review by bascoe10 — APPROVED** [review]

**Comment by ernestkamara** [comment]
FYI: Can you please summarise what has been done and what new plugins were added in your PR description.
I find this very useful before reviewing PRs.
Just my opinion.

**Comment by J4yM1l** [comment]
Added a form where admin can be able to add test center details to database such as address, city with regional location if necessary and phone number. In other to access the form, type "/admin" on url.
FYI: No additional plugins were added.

File: src/components/admin-form.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        this.state = {
+            city: '',
+            location: '',
+            address: '',
+            phone: '',
+            date: Date.now()
+        };
+        // console.log('Porps:', props);
+
+        // this.handleChange = this.handleChange.bind(this);
```
Comment (ernestkamara) [inline src/components/admin-form.js]: Remove commented lines if not needed

File: src/components/admin-form.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        
+           if (errorCode === 'auth/operation-not-allowed') {
+             //alert('You must enable Anonymous auth in the Firebase Console.');
+           } else {
+            alert('Server down, please try again later!');
+             console.log(error);
+
+           }
+         }).then(function(crenditial) {
+            console.log(crenditial.user.uid);
```
Comment (ernestkamara) [inline src/components/admin-form.js]: Remove this, we don’t want to expose this I’d

**Review by ernestkamara — CHANGES_REQUESTED** [review]
See comments.

**Comment by ernestkamara** [comment]
Submitting this form requires a valid admin account in my opinion . Deleting this submission would be hard since the data is being stored anonymously. Can you elaborate the use case here or on slack? Just making sure we need it and we’ll thought off.

**Review by ernestkamara — APPROVED** [review]

**Comment by J4yM1l** [comment]
@ernestkamara, @bascoe10 -Requested changes has been made to the admin form plus a new input field was also added (hospitalName) which will be updated on the test-center table soon. In addition, form input is now validated for empty string or incorrect number format.

File: src/components/test-centers.js [hunk 3]
```diff
@@ -20,27 +19,35 @@
...
     
     // eslint-disable-next-line no-useless-constructor
     constructor(props){
         super(props)
+        this.state = {
+          rows: [],
+        }
+    }
+    componentDidMount(){
+      const context = this
```
Comment (ernestkamara) [inline src/components/test-centers.js:33]: Is this an update to an existing document or an insertion? I haven't run this PR yet.

**Review by ernestkamara — APPROVED** [review]
Works!

File: src/components/test-centers.js (hunk 3 above)
Comment (J4yM1l) [reply src/components/test-centers.js:33]: Actually, neither, but it's for fetching. Those changes should have reflected on the test-center pr not here, maybe I might make a change on that later. Look at test center pr for full detail.

**Pull request closed without merging**